### ISSUE TITLE
Remove `ToyWorld` and `ToyAccount`

### ISSUE THREAD

**Issue by DavePearce** [Consensys/besu-sequencer-plugins#1048]
Once #1045 is merged then we should no longer have any requirement for `ToyWorld` and `ToyAccount` (plus perhaps other things) so they could be removed.

As part of this, we should tidy up the changes made to `ToyExecutionEnvironment`.

@gauravahuja FYI

**Comment by OlivierBBB** [comment]
@DavePearce is #1045 the issue you merged into #1027 ? If so I will be merging it later once I get a review from @letypequividelespoubelles.

**Comment by DavePearce** [comment]
Yeah thats right. I'll wait for you then will fix this up and put in whats left
